server/server_app.js

```diff
@@ -423,6 +425,62 @@ app.get(
   }
 )
 
+/**
+ * Route used for Create/Update auth token
+ * If no token storage is already linked with the user, a new one is created
+ * body.google The Google auth token (Set to '' to remove it)
+ * body.discord The Discord auth token (Set to '' to remove it)
+ * Route protected by a JWT token
+ */
+app.post(
```

Comment:
There is an better way to do this.

You can fetch the user with its Token. Then, if it has a Token, update it, else update the user by creating a token.

Reply:
updated